src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,49 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_USER_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        return (dataIsNullCheck(user)
+                && ageCheck(user)
+                && loginCheck(user)
+                && passwordCheck(user)) ? storageDao.add(user) : null;
+    }
```

Comment:
is it possible that this method will return `null`?
looks like each method throws an exception or returns true

Reply:
can I make all of this checking methods void?